classes/class-wc-connect-nux.php

```diff
@@ -83,5 +94,168 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 			);
 			return $pointers;
 		}
+
+		public function get_jetpack_install_status() {
+			$status = Jetpack_Install_Status::UNINSTALLED;
+
+			if( 0 === validate_plugin( 'jetpack/jetpack.php' ) ) {
+				$status = Jetpack_Install_Status::INSTALLED;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
+				// installed, but not activated
+				return $status;
+			}
+
+			if ( defined( 'JETPACK_DEV_DEBUG' ) ) {
```

Comment:
So if Jetpack is not unstalled, we're going to be returning `Jetpack_Install_statis::DEV`. Is this what we want to return?